### ISSUE TITLE
Accessibility: Usability: Narrator: Narrator announces wrong item numbers in the "Tweets" list

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#7]
#A11yUsable
#Wipro
#vscode-website
#Firefox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

Firefox Version:
50.1.0

**Repro Steps:**
1. Open https://code.visualstudio.com/ in EDGE.
2. Navigate to the Tweets list available above the "Meet IntelliSense" Header (with Caps Lock + --> )
3. Observe the Narrator announcement

**Actual Result:**
The Narrator announces the Tweets list selection as "Selected, selection contains 0 items"
[Inspect tree displays that there are many items under the list](refer screenshot)

Inspect Behavior:
Name: ""
ControlType: UIA_ListControlTypeId (0xC358)
LocalizedControlType: "list"

**Expected Result:**
The Narrator should announce the correct number of items present in the "Tweets" list.

**User Impact:**
Users who rely on screen readers will be  updated with wrong data and they will be misguided by these type of wrong information

![narrator announces as selection contains 0 items](https://cloud.githubusercontent.com/assets/24825441/21674158/844d08c2-d352-11e6-9288-e72041d8d8cf.png)
![inspect behaviour screenshot 2](https://cloud.githubusercontent.com/assets/24825441/21674169/8fb3cee4-d352-11e6-9513-eac994d4ce28.png)
![narrator buddy screenshot 2](https://cloud.githubusercontent.com/assets/24825441/21674176/99104864-d352-11e6-91ab-3548e8d198b4.png)
![nvda not reading number of items in list](https://cloud.githubusercontent.com/assets/24825441/21674198/b112a790-d352-11e6-9740-4c13cae45de7.png)

**Comment by wiproaccess** [comment]
The issue is verified hence adding the label 'closed' and closing the bug.
